rasa/core/lock_store.py

```diff
@@ -79,7 +82,9 @@ def issue_ticket(
         logger.debug(f"Issuing ticket for conversation '{conversation_id}'.")
         try:
             lock = self.get_or_create_lock(conversation_id)
-            ticket = lock.issue_ticket(lock_lifetime)
+            lock.remove_expired_tickets()
+            ticket_number = self.increment_ticket_number(lock)
+            ticket = lock.issue_ticket(lock_lifetime, ticket_number)
```

Comment:
I see that `remove_expired_tickets` is called in several places; in this method it means the `remove_expired_tickets` is called three times (here, by `issue_ticket`, and `save_lock`).
Is this necessary, or can it be done once only?